### ISSUE TITLE
Fetch integration is missing

### ISSUE THREAD

**Issue by annevk** [w3c/server-timing#55]
It's unclear when this header is processed.

**Comment by yoavweiss** [comment]
@annevk what's the best way to integrate it?
It seems like such integration would be trivial once [ResourceTiming is defined in terms of Fetch](https://github.com/w3c/resource-timing/issues/39), but that's a slightly bigger project.

Generally, do you think Fetch should be aware and call into RT/ST processing models, or should Fetch provide a hook and have these specs "register for a callback" from Fetch?

**Comment by annevk** [comment]
Yeah, I guess you need resource timing defined as this would layer on that infrastructure.

I'm wondering if we add the timing information directly to fetch, if this could all be on top of fetch groups (which are ill-defined, but still). It seems the timing information (though not necessarily that of "server timing", as that comes from a header on the response) is primitive of sorts we'll need to wave through the various algorithms and expose appropriately.

**Comment by GBailey7** [comment]
The timing is lagging for the fetch

**Comment by noamr** [comment]
See https://github.com/w3c/server-timing/pull/87
